.env.production

```diff
@@ -1,7 +1,8 @@
 # This enviroment file is choosen by the build process when using the 'release' variant/scheme
 
 ENVIRONMENT=PROD
-API_URL_PREFIX='https://app-backend.k8s.test.cd.teamdigitale.it'
+#API_URL_PREFIX='https://app-backend.k8s.test.cd.teamdigitale.it'
+API_URL_PREFIX='https://app-backend-rc.k8s.test.cd.teamdigitale.it'
```

Comment:
Please check this.

Reply:
this is fine for the draft PR, the backend logic has been deployed on the -rc backend instance